### PR TITLE
Updated README and added a CONTRIBUTING

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+We welcome pull requests from anyone.
+
+Fork, then clone the repo:
+
+    git clone git@github.com:your-username/aaas.git
```
Comment (RealOrangeOne) [inline CONTRIBUTING.md]: I'm not sure we need to tell them how to clone repos. Especially if they arent using SSH

File: README.md [hunk 2]
```diff
@@ -1,18 +1,41 @@
 # aaas
-Ainsley As A Service
+Ainsley As A Service. It's [placekittens](https://placekiteens.com) but for Ainsley Harriot
```
Comment (RealOrangeOne) [inline README.md]: _It's placekittens, but for Ainsley Harriot_? (note the comma)

File: README.md [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
 - NodeJS
 - NVM
 
-## Setup
+### Setup
+To setup the the project locally run the following commands:
+
     ./scripts/build
+
+This command setup the virtual environment, get the required models, and create the database
```
Comment (RealOrangeOne) [inline README.md]: This doesnt actually create the database, that has to be done seperately, but I guess it could go into the script?

**Review by RealOrangeOne — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sedders123) [reply README.md]: I can just add it as an extra step for now but its something to think about

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**